std_msgs/Header header

float32[] xGlbPath

float32[] yGlbPath

std_msgs/Float32MultiArray xLocPath

std_msgs/Float32MultiArray yLocPath
